### ISSUE TITLE
can't find https://jeff-zucker.github.io/solid-file-client/solid-file-client.js

### ISSUE THREAD

**Issue by scenaristeur** [jeff-zucker/solid-content-manager#12]
Hi Jeff, 
after git clone, trying to open index.html in firefox gives me this error : 

GEThttps://jeff-zucker.github.io/solid-file-client/solid-file-client.js
[HTTP/2.0 404 Not Found 18ms]

```
GEThttps://solid.github.io/releases/rdflib.js/rdflib-0.12.2.min.js
[HTTP/2.0 200 OK 69ms]
GEThttps://solid.github.io/solid-auth-client/dist/solid-auth-client.bundle.js
[HTTP/2.0 200 OK 69ms]
GEThttps://cdn.jsdelivr.net/npm/vue/dist/vue.min.js
[HTTP/2.0 200 OK 30ms]
GEThttps://cdnjs.cloudflare.com/ajax/libs/ace/1.4.1/ace.js
[HTTP/2.0 200 OK 26ms]
GEThttps://jeff-zucker.github.io/solid-file-client/solid-file-client.js
[HTTP/2.0 404 Not Found 20ms]
GEThttps://jeff-zucker.github.io/solid-file-client/solid-file-client.js
[HTTP/2.0 404 Not Found 18ms]
Échec du chargement pour l’élément <script> dont la source est « https://jeff-zucker.github.io/solid-file-client/solid-file-client.js ». index.html:19:1
ReferenceError: SolidFileClient is not defined[En savoir plus]

```

I've tried with https://cdn.jsdelivr.net/npm/solid-file-client/dist/browser/solid-file-client.bundle.js
 as mentioned https://jeff-zucker.github.io/solid-file-client/ 

then I got this error 
``` 
GEThttps://solid.github.io/releases/rdflib.js/rdflib-0.12.2.min.js
[HTTP/1.1 200 Connection established 136ms]
GEThttps://solid.github.io/solid-auth-client/dist/solid-auth-client.bundle.js
[HTTP/2.0 200 OK 118ms]
GEThttps://cdn.jsdelivr.net/npm/vue/dist/vue.min.js
[HTTP/1.1 200 Connection established 66ms]
GEThttps://cdnjs.cloudflare.com/ajax/libs/ace/1.4.1/ace.js
[HTTP/1.1 200 Connection established 62ms]
GEThttps://cdn.jsdelivr.net/npm/solid-file-client/dist/browser/solid-file-client.bundle.js
[HTTP/2.0 200 OK 1712ms]
Caution: multiple versions of solid-auth-client active. index.js:111
TypeError: SolidFileClient is not a constructor
```
any solution ?
txs

**Comment by jeff-zucker** [comment]
Oh, sorry, I didn't see this until now, need to change my github notifications.  The problem is that I have never switched solid-ide over to solid-file-client, it still uses the original version at https://github.com/jeff-zucker/solid-ide/blob/master/solid-file-client.js which is the OLD version.  I have been busy with other things and hope to get back soonish to change this.  So you can either point there instead of in the solid-file-client repo, or else you can start using the new solid-file-client that is in the solid-file-client repo.  There are several changes between the two, and the one that bit you is the name of the solid-file-client object.  See https://github.com/jeff-zucker/solid-file-client/blob/master/changes.log for a list of the changes between the original and the new.   Another thing (only in the new version) - it already bundles solid-auth-client and rdflib in it, so there is no need to explicitly import them in your script tag.

**Comment by bourgeoa** [comment]
closed with PR #14
